### ISSUE TITLE
Getting MTP to work on Mac

### ISSUE THREAD

**Issue by motobai** [playfultechnology/audio-guestbook#70]
Hi everyone,
I finally got my audio guestbook to work - except for the MTP feature.
I'm working with a Teensy 4.0 and I'm on Mac OS.
When compiling, I get the warning

> "/Users/username/Documents/Arduino/libraries/MTP_Teensy/src/MTP_Teensy.cpp: In member function 'uint32_t MTP_class::GetObjectInfo(MTP_class::MTPContainer&)':
> /Users/username/Documents/Arduino/libraries/MTP_Teensy/src/MTP_Teensy.cpp:523:34: warning: unknown option after '#pragma GCC diagnostic' kind [-Wpragmas]
>    #pragma GCC diagnostic ignored "-Wformat-truncation" /* Or "-Wformat-truncation" */"

Nevertheless, the code compiles successfully and the guestbook works, but I do not see a Teensy external drive when connected via USB, nothing in disk utility either.

Serial Monitor says

> &&&&& Dump MTPStorage Loop Data &&&&&
	Callback Data:		0	0x0	0
		1	0xcde5	1
		2	0x0	0
		3	0x0	0
		4	0x0	0
	File Systems:
		0	0x200035cc	1

My SD Card is formatted in exFAT.

Am I missing something?

**Comment by motobai** [comment]
So I figured out it's a Mac problem (surprise). Mac computers apparently do not natively support MTP.
I tried a PC running Windows, no problem there, the Teensy with its SD card mounts as a drive.

I tried different Mac softwares like Android File Transfer, SyncMate and MacDroid, but they didn't work. 
[MTP for Mac](https://www.hyperintegrate.com/products/mtp-for-mac) partially works, but it's read only, no way to write or delete files on the SD card via MTP.
